SPECS/cloud-hypervisor/cloud-hypervisor.spec

```diff
@@ -1,18 +1,17 @@
 Summary:        A Rust-VMM based cloud hypervisor from Intel
 Name:           cloud-hypervisor
 Version:        22.0
-Release:        2%{?dist}
+Release:        3%{?dist}
 License:        ASL 2.0 or BSD
 URL:            https://github.com/cloud-hypervisor/cloud-hypervisor
 Group:          Development/Tools
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:       %{url}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
-# Note: the %%{name}-%%{version}-cargo.tar.gz file contains a cache created by capturing the contents downloaded into $CARGO_HOME.
```

Comment:
I think, we should not remove this comment.
It was probably intentional.

Reply:
@mfrw , Thank you for the review and also for suggesting the "cargo vendor" method. I have addressed your comments. Please take a look . 
Also added a similar comment for %{name}-%{version}-vendor-%{release}.tar.gz